spec/features/openid_connect/phishing_resistant_required_spec.rb

```diff
@@ -19,27 +19,59 @@
 
     context 'user has aal3 auth configured' do
       it 'sends user to authenticate with AAL3 auth' do
-        sign_in_before_2fa(user_with_aal3_2fa)
+        sign_in_before_2fa(user_with_phishing_resistant_2fa)
 
         visit_idp_from_ial1_oidc_sp_requesting_aal3(prompt: 'select_account')
```

Comment:
**nitpick:** can we rename these methods (and any other helper methods) to not use `aal3`?

Reply:
actually if these are actually testing ACR values of `aal/3` then we can leave them for now (since we'll ultimately remove them) - but the `context` on line 20 could be changed.

Reply:
updated in d5c687bcb